Watchman.Engine.IntegrationTests/AlarmFinderTests.cs

```diff
@@ -32,8 +32,7 @@ public async Task TestCount()
 
         private static IAlarmFinder MakeAlarmFinder()
         {
-            var cloudwatch = new AmazonCloudWatchClient(CredentialsReader.GetCredentials(),
-                new AmazonCloudWatchConfig {RegionEndpoint = RegionEndpoint.EUWest1});
+            var cloudwatch = new AmazonCloudWatchClient(RegionEndpoint.EUWest1);
```

Comment:
This change is ok becuase the SDK will read from the config file anyway by itself?

Reply:
that's right, was causing failures on my local machine